zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java

```diff
@@ -322,7 +319,8 @@ public boolean runParagraph(String noteId,
 
     try {
       notebook.saveNote(note, context.getAutheInfo());
-      boolean result = note.run(p.getId(), blocking);
+//      boolean result = note.run(p.getId(), blocking);
+      boolean result = note.run(p.getId(), blocking, context.getAutheInfo().getUser());//fix for ZEPPELIN-3065 -- shoaib.rehman
```

Comment:
`//fix for ZEPPELIN-3065 -- shoaib.rehman`
I think this comment can be deleted because it doesn't make sense.

Reply:
@srehmann , Please delete `//      boolean result = note.run(p.getId(), blocking);`